src/GenerateIndex.scala

```diff
@@ -1,7 +1,7 @@
 //> using scala 3
-//> using dep com.softwaremill.sttp.client3::core:3.9.0
-//> using dep com.lihaoyi::ujson:3.1.3
-//> using dep com.lihaoyi::os-lib:0.9.1
+//> using dep com.softwaremill.sttp.client3::core::3.9.3
+//> using dep com.lihaoyi::ujson::3.2.0
+//> using dep com.lihaoyi::os-lib::0.9.3
```

Comment:
```suggestion
//> using dep com.softwaremill.sttp.client3::core:3.9.3
//> using dep com.lihaoyi::ujson:3.2.0
//> using dep com.lihaoyi::os-lib:0.9.3
```

We're not actually needing any cross platform stuff here are we?